### PR TITLE
Conference page: Create Home and About pages

### PR DESCRIPTION
**Pull request by helio3197**
### Capstone Project: Added Home and About pages
The pages Home and About for the Conference website were created applying the good practices for HTML, CSS, and JavaScript and a correct workflow.
#### Highlighted Features:
- **Responsiveness:** Both pages are completely responsive and can adapt themselves to every kind of screen and devices.
- **Efficiency:** Speakers' information is stored in a JavaScript object and displayed upon reloading the page, new speakers can easily be added just by appending their information on the object.
#### Technologies implemented:
- **Bootstrap:** This CSS framework allows building the page faster, keeping a good layout and full responsiveness. All the sections for the pages were created using Bootstrap.
- **Sass:** This CSS pre-processor was used to implement the styles to the pages in a more efficient way.
- **JavaScript:** Some functionalities of the pages were created using JavaScript like the dynamically created Speakers section and the mobile menu.
#### Demo video:
https://www.loom.com/share/7d37cce319704acfb7191d54f6fbe006

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+              <button type="button" class="btn-close text-reset d-md-none" data-bs-dismiss="offcanvas" id="closeMenu"></button>
+            </div>
+            <div class="navbar-nav navbar-items gap-md-4">
+              <a href="./about.html" class="nav-item nav-link current-page">About</a>
+              <a href="#" class="nav-item nav-link">Program</a>
+              <a href="#" class="nav-item nav-link">Sponsors</a>
+              <a href="#" class="nav-item nav-link">News</a>
+              <div class="button-join">
+                <a href="#">Join WBS</a>
+              </div>
```
Comment (micheaol) [inline about.html]: Please ensure that It's possible to navigate between the pages, there's a link from the `Home page` to the `About page` and there should be a link from the `About page` to `the Home page`. Please update your `nav menu when you navigate between pages: If you are on `Home page` there should be `nav menu to `About page` same with `About page`. Please make this adjustment on ALL window sizes.


Nav

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @helio3197 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 


## Suggested changes

- _Please check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: about.html (hunk 1 above)
Comment (helio3197) [reply about.html]: Hi @micheaol , according with the project requirements: "The home page should have a link in the menu to the about page.
**The logo in the header links to the home page.**" The logo in the navbar currently links to the home page.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        <div class="container-md navbar navbar-expand-md navbar-light">
+          <button type="button" class="navbar-toggler border-0" data-bs-toggle="offcanvas" data-bs-target="#navbarSlide">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <nav class="offcanvas offcanvas-start navbar-collapse border-0" tabindex="-1" id="navbarSlide">
+            <div class="offcanvas-header align-self-stretch  me-md-auto p-md-0 logo-container">
+              <div class="offcanvas-title">
+                <a href="./index.html">
+                  <img src="images/WBS-logo.png" alt="wbs logo" class="logo">
+                </a>
```
Comment (helio3197) [inline about.html:36]: There is the link to the homepage

File: about.html (hunk 1 above)
Comment (helio3197) [reply about.html]: Can you please elaborate more your suggestion?

Comment (micheaol) [reply about.html]: @helio3197,
According to the project requirement: If you are on the `Home page` there should be a `nav Link` to `About page` and if you are on the `About page` there should be a `nav Link` to `Home page`.   You can add `Home` to your `navbar links`  in  your `About page`

File: about.html (hunk 2 above)
Comment (micheaol) [reply about.html:36]: @helio3197,
According to the project requirement: If you are on the `Home page` there should be a `nav Link` to `About page` and if you are on the `About page` there should be a `nav Link` to `Home page`. You can add `Home` to your `navbar links` in  your `About page`

Comment (helio3197) [reply about.html:36]: Thank you for your feedback, I understand now.

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
Hi @helio3197 , 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

well-1
Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

```

**Pull request merged**